### PR TITLE
Switching prcheck to new tests.

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
Switch from using old tests from che-functional-tests to new test suite.

### CODE REVIEW

**Comment by Katka92** [comment]
[test]

**Comment by Katka92** [comment]
new image was loaded, running again tests
[test]

**Comment by ScrewTSW** [comment]
LGTM ^.=.^

File: functional-tests/devscripts/run_tests.sh [hunk 1]
```diff
@@ -48,29 +48,39 @@ while getopts "hu:p:m:o:r:" opt; do
...
+	           -e "RHCHE_HOST_PROTOCOL=http" \
+	           -e "RHCHE_PORT=80" \
+	           -e "RHCHE_OPENSHIFT_TOKEN_URL=https://sso.prod-preview.openshift.io/auth/realms/fabric8/broker" \
+	           -e "TEST_SUITE=prcheck.xml" \
            quay.io/openshiftio/rhchestage-rh-che-functional-tests-dep
     RESULT=$?
 else
-	echo "Running test with user $USERNAME against prod-preview environment."
-	CHE_OSIO_AUTH_ENDPOINT="https://auth.prod-preview.openshift.io"
+	if [[ -z $USERNAME || -z $PASSWORD || -z $EMAIL || -z $HOST_URL ]]; then
```
Comment (rhopp) [inline functional-tests/devscripts/run_tests.sh]: The indentation is quite off, making this hard to read...

**Comment by rhopp** [comment]
I have one general comment.
I think it would be better to have `rh-che/functional-tests/devscripts/run_tests.sh` as a "general" script and move the pr check logic to some of the `.ci/...` scripts (mainly setting correct value to required variables such as `$USERNAME`, `$PASSWORD`, `$HOST_URL` etc.

**Comment by Katka92** [comment]
[test]

**Pull request merged**